src/components/WebexActivity/README.md

```diff
@@ -16,18 +16,20 @@ To see all the different possible states of the Webex Activity component, you ca
 
 ## Embed
 
-1. Create a component adapters object from which the data will be retrieved (See [adapters](../../adapters)). For instance:
+1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
```

Comment:
Should we list the required data adapters that this component needs? As our webex adapter grows with sub-adapters, there will be data that this component doesn't need and isn't required to render properly.

Reply:
I thought about that, but I think which sub-adapter the component uses not relevant to a consumer of the components. Ideally, we want to incentivize our users to use the complete adapter. 
Sub-adapters are meant more for developers that want to create their own adapters and our sanity 😅

Reply:
Understood.